cfme/v2v/migrations.py

```diff
@@ -363,17 +372,52 @@ def plan_in_progress(self, plan_name):
             return False
 
 
+class MigrationDashboardView59z(MigrationDashboardView):
+    infra_mapping_list = InfraMappingList("infra-mappings-list-view")
+
+    @property
+    def is_displayed(self):
+        return (self.navigation.currently_selected == ['Compute', 'Migration'] and
+            (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0) and
+            (len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0))
+
+
+class InfrastructureMappingView(BaseLoggedInPage):
+    infra_mapping_list = InfraMappingList("infra-mappings-list-view")
+    create_infrastructure_mapping = Text(locator='(//a|//button)'
+                                                 '[text()="Create Infrastructure Mapping"]')
+    configure_providers = Text(locator='//a[text()="Configure Providers"]')
+    paginator_view = View.include(InfrastructureMappingsPaginatorPane, use_parent=True)
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    # Used for Ascending/Descending sort
+    sort_order = Text(".//button[./span[contains(@class,'sort-direction')]]")
+    # Used to select filter_by 'Name' or 'Description'
+    filter_by_dropdown = SelectorDropdown('id', 'filterFieldTypeMenu')
+    # USed to select sort by options like 'Name', 'Number of Associated Plans'
+    sort_by_dropdown = SelectorDropdown('id', 'sortTypeMenu')
+
+    @property
+    def is_displayed(self):
+        # TODO: Remove 1st condition, once /manageiq-v2v/issues/726 fix is backported to 510z
+        return ((self.navigation.currently_selected ==
+            ['Compute', 'Migration'] or self.navigation.currently_selected ==
+            ['Compute', 'Migration', 'Infrastructure Mappings']) and
+            len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0 and
```

Comment:
consider expressing this as `not self.browser.elements()`, which is returning an empty list on no locator match.